### ISSUE TITLE
Subtlety - Test Case Results

### ISSUE THREAD

**Issue by cc-prj-review-alf** [ajohnson10209/CodeacademyBuildWebAPICapstone#5]
So great job here, 87/89 test are passing. This put you at 97.75% passing for the tests. Great job here! The goal, of course, if you are out in the field is to try to pass 100% of the time. However, for the context of this assignment, you did meet the expectations for this priority.

**Comment by ajohnson10209** [comment]
Yeah, I got stumped with the default field not working. Especially since when I was doing a console log of a record from the employee table, is_employed was returning 1. The other two were issues with returning the array which I didn't understand either.
